lib/ui.js

```diff
@@ -2,18 +2,40 @@
 const path = require('path')
 const serve = require('serve-static')
 
-module.exports.serveRedoc = function serveRedoc (documentUrl) {
+module.exports.serveRedoc = function serveRedoc (documentUrl, opts) {
+  const toKebabCase = (string) =>
+    string
+      .replace(/([a-z])([A-Z])/g, '$1-$2')
+      .replace(/[\s_]+/g, '-')
+      .toLowerCase()
+  const options = {}
+  Object.keys(opts).forEach((key) => {
+    if (!['coerce', 'htmlui', 'basePath'].includes(key)) {
+      options[toKebabCase(key)] = opts[key]
+    }
+  })
```

Comment:
If you use `reduce` here you don't even need to enumerate the keys again on line 22. Not blocking, but just wanted to mention it.

Reply:
Reduce hides a lot of complexity, so I do not use it often, but thanks for letting me know.

Reply:
You and a lot of other people agree. I hate some parts of that api, especially when you don't pass an initial value, but it really is nice for situations like this. Anyway, not worried about doing this way either.